src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -404,6 +405,7 @@ export const View = (props: Props) => {
   };
 
   const showCsatSurvey =
+    isPlusUser &&
```

Comment:
This is the actual needed change right? Per above I'd just continue calling it `premium` here just for now, and rename this holistically as a separate changeset.

Reply:
Updated in [0322800](https://github.com/mozilla/blurts-server/pull/4516/commits/032280010ec31030dea3b29754b02eb102a4dcde).